### ISSUE TITLE
[Firefox] It breaks YouTube's hold to play at 2x speed feature

### ISSUE THREAD

**Issue by lilydjwg** [polywock/globalSpeed#540]
Without this extension, holding left mouse button on a YouTube video, it will play at 2x speed. But with this extension, it only plays at 2x for a fraction of second and then continue with normal speed.

**Comment by polywock** [comment]
Global Speed is very controlling about the speed. It's very hard to determine if the user intended to change the speed or the website is doing it on it's own. That's why Global Speed requires speed changes to through the extension itself. 

Personally, I use shortcuts to accomplish same thing. You can create a shortcut that sets speed to 4x. The first time you press it, speed will go to 4x. Second time you press it, it will go back to the original speed.

**Comment by lilydjwg** [comment]
Thanks, a shortcut works, except it takes some time to figure a good shortcut as I have so many of them from different places.

Can't global speed just let the website change speed? There is an option called "ghost mode". I thought it meant that.

**Comment by polywock** [comment]
Ghost mode actually blocks the website from changing the speed at all 😆 

Are you Chrome?

**Comment by polywock** [comment]
There might be a workaround in the next version I'll update for Firefox. It might take a month or so.

**Comment by Zqiancreator** [comment]
Has this problem solved now?
Should I still add a new shortcut or the newest version of Global Speed have had a potential solution now?

**Comment by yarub123** [comment]
Yeah the holding spacebar (or mouse left click) on a youtube video to make it 2x speed does not work. Global speed seems to override the behavior.

**Comment by polywock** [comment]
Hello! The new update will include a "Press and hold on videos to accelerate" feature. By default, it will be 2x but users will be able to customize the acceleration.  This should work on any website, not just Youtube. 

Update should be available within a week.
